tickets/views.py

```diff
@@ -403,12 +404,21 @@ def moderation_assigned(request, user_id):
 
             if action == "Approve":
                 tickets.update(status=TICKET_STATUS_CLOSED)
-                explicit = mod_sound_form.cleaned_data.get("is_explicit")
-                Sound.objects.filter(ticket__in=tickets).update(
-                        is_index_dirty=True,
-                        moderation_state='OK',
-                        moderation_date=datetime.datetime.now(),
-                        is_explicit=explicit)
+                sounds_update_params = {
+                    'is_index_dirty': True,
+                    'moderation_state': 'OK',
+                    'moderation_date': datetime.datetime.now()
+                }
+                is_explicit_choice_key = mod_sound_form.cleaned_data.get("is_explicit")
+                if is_explicit_choice_key == IS_EXPLICIT_ADD_FLAG_KEY:
+                    sounds_update_params['is_explicit'] = True
+                elif is_explicit_choice_key == IS_EXPLICIT_REMOVE_FLAG_KEY:
+                    sounds_update_params['is_explicit'] = False
+                elif is_explicit_choice_key == IS_EXPLICIT_KEEP_USER_PREFERENCE_KEY:
```

Comment:
I guess you're adding this to make it explicit what happens when the key is set to this value, but I wonder if we could just have the following comment instead of the elif/pass